ADR/0036-trusted-artifacts.md

```diff
@@ -0,0 +1,47 @@
+# 36. Trusted Artifacts
+
+Date: 2024-06-11
+
+## Status
+
+Accepted
+
+## Context
+
+One of the properties of Konflux is that users should be allowed to include their own Tekton Tasks
+in a build Pipeline, e.g. to execute unit tests, without invalidating the build process. This is
+distinct from other build systems where a rigid process prevents users from applying such
+customizations. To support this, Konflux build Pipelines use Trusted Artifacts to securely share
+data between Tasks. Enterprise Contract is then responsible for verifying that Trusted Artifacts
```

Comment:
```suggestion
files between Tasks. Enterprise Contract is then responsible for verifying that Trusted Artifacts
```

Reply:
Good to be specific. Also modified other occurrences.